web-wallet/src/lib/components/Send/Send.svelte

```diff
@@ -120,10 +133,67 @@
         href: "/dashboard",
         isAnchor: true,
       }}
-      nextButton={{ disabled: isNextButtonDisabled }}
+      nextButton={{
+        action: () => {
+          activeStep = 1;
+        },
+        disabled: !addressValidationResult.isValid,
+      }}
     >
       <div in:fade|global class="operation__send">
-        <ContractStatusesList items={statuses} />
+        <div class="operation__send-amount operation__space-between">
```

Comment:
Here, `__space-between`  is used as if it's an element – which it is not. Instead, this should be a modifier.

Reply:
I agree, however this has been in the component for [a long time ](https://github.com/dusk-network/rusk/blame/master/web-wallet/src/lib/components/Send/Send.svelte#L311).  I will open a separate issue.